SeaPkg/Tools/GenSeaArtifacts/test_aux/src/efi.rs

```diff
@@ -12,6 +12,11 @@ use std::{
 
 use r_efi::efi::Status;
 
+const WIDTH: usize = 8usize * core::mem::size_of::<Status>();
+const WARNING_MASK: usize = 0x0 << (WIDTH - 8);

```

Comment:
nit: `STATUS_WIDTH` is more precise.

```suggestion
const STATUS_WIDTH: usize = 8usize * core::mem::size_of::<Status>();
const WARNING_MASK: usize = 0x0 << (STATUS_WIDTH - 8);
```